homework-submission/main.js

```diff
@@ -85,7 +85,7 @@ const TodoList = ({ todoArr, liClickHandler, buttonClickHandler, clickBehaviour
 };
 
 class Main extends React.Component {
-  state = { todoArr, behaviour: 'mark' };
+  state = { todoArr: [...todoArr], behaviour: 'mark' };
```

Comment:
You are still using `todoArr` here in your Main component, even though this is a global variable. What @shrynx meant is to rewrite your code in such a way that you can render your Main component the following way:
```jsx
ReactDOM.render(<Main todoArr={todoArr} />, root);
```